tasks/postgres-database-extensions.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "Install required packages for 'postgres_ext' ansible module"
+  apt:
+    name: "{{ item }}"
+    state: present
+  loop:
+    - libpq-dev
+    - python-psycopg2
+
+- name: Install system dependencies of extensions
+  apt:
+    name: "{{ item }}"
```

Comment:
Since ansible 2.7 will now do an apt call per item coul you pass in name the list ?

Reply:
:ok_hand: I did not now that `apt.name` supported a list directly. Nice!